tools/pontoon/translate.md

```diff
@@ -1,36 +1,36 @@
-# Translating in Pontoon
+## Translating in Pontoon
 
 * [Translating Strings](#translating-strings)
   * [Submit a Suggestion](#submit-a-suggestion)
   * [Review a Suggestion](#review-a-suggestion)
   * [Submit a Translation](#submit-a-translation-directly)
-* [Translating in the In-context Editor](#translating-in-the-wysiwyg-editor)
+* [Translating using the the In-context Editor](#translating-using-the-In-context-editor)
 
-## Translating Strings
+### Translating Strings
 
-Three string actions can be performed: submit a suggestion, review a suggestion, and submit a translation directly. The level to which a user can translate a string is dependent upon their user role (Read about user roles [here](user.md)).
+Three string actions can be performed: submit a suggestion, review a suggestion, and submit a translation directly. The level to which a user can translate a string depends on their user role (Read about [user roles](users.md)).
```

Comment:
Sorry, meant `… can translate a string depends on their [user role](users.md)`. I think it's clear enough without an explicit invite to read